stix_shifter/stix_transmission/src/modules/synchronous_dummy/api_client.py

```diff
@@ -0,0 +1,27 @@
+from ..utils.RestApiClient import RestApiClient
+
+
+class APIClient():
+
+    def __init__(self, connection, configuration):
+        # auth = configuration.get('auth')
```

Comment:
we should put comment the reason of commenting out this block of code. i know its obvious but since its a public repo some people may miss obvious things :)